exporter/awsemfexporter/emf_exporter.go

```diff
@@ -133,14 +141,20 @@ func (emf *emfExporter) pushMetricsData(_ context.Context, md pdata.Metrics) (dr
 				err = wrapErrorIfBadRequest(&returnError)
 				return
 			}
-			returnError = pusher.ForceFlush()
-			if returnError != nil {
-				err = wrapErrorIfBadRequest(&returnError)
-				return
-			}
 		}
 	}
 
+	for _, pusher := range emf.listPushers() {
+		returnError := pusher.ForceFlush()
+		if returnError != nil {
```

Comment:
Thanks for catching the performance issue here. 
Could you combine this code block with the code in `func (emf *emfExporter) Shutdown()`?

Reply:
Done